templates/devspace.yaml.tpl

```
@@ -10,113 +12,79 @@ vars:
   # It's probably not the best approach, but it works well.
-  - name: DEVENV_DEPLOY_APPNAME
+  DEVENV_DEPLOY_APPNAME:
     source: env
     default: {{ .Config.Name }}
 
-  - name: DEVENV_DEPLOY_LABELS
-    value:
-      app: ${DEVENV_DEPLOY_APPNAME}
-
-  - name: DEV_CONTAINER_EXECUTABLE
-    value: ${DEVENV_DEPLOY_APPNAME}
+  DEV_CONTAINER_EXECUTABLE: ${DEVENV_DEPLOY_APPNAME}
   # ================================================
 
-  - name: DEPLOY_TO_DEV_VERSION
+  DEPLOY_TO_DEV_VERSION:
     source: env
     default: "latest"
-  - name: DEVENV_DEPLOY_VERSION
+  DEVENV_DEPLOY_VERSION:
     source: env
     default: "latest"
-  - name: DEVENV_DEPLOY_IMAGE_SOURCE
+  DEVENV_DEPLOY_IMAGE_SOURCE:
     source: env
     default: local
-  - name: DEVENV_DEPLOY_DEV_IMAGE_REGISTRY
+  DEVENV_DEPLOY_DEV_IMAGE_REGISTRY:
     source: env
     default: devenv.local
-  - name: DEVENV_DEPLOY_BOX_IMAGE_REGISTRY
+  DEVENV_DEPLOY_BOX_IMAGE_REGISTRY:
     source: env
     default: gcr.io/outreach-docker
-  - name: DEVENV_DEPLOY_IMAGE_REGISTRY
+  DEVENV_DEPLOY_IMAGE_REGISTRY:
     source: env
     default: ${DEVENV_DEPLOY_BOX_IMAGE_REGISTRY}
-  - name: DEVENV_TYPE
+  DEVENV_TYPE:
     source: env
-  - name: DEVENV_DEV_TERMINAL
+  DEVENV_DEV_TERMINAL:
     source: env
     default: "false"
-  - name: DEVENV_DEV_SKIP_PORTFORWARDING
+  DEVENV_DEV_SKIP_PORTFORWARDING:
     source: env
     default: "false"
-  - name: DEVENV_DEV_DEPLOYMENT_PROFILE
+  DEVENV_DEV_DEPLOYMENT_PROFILE:
     source: env
     default: deployment__{{ .Config.Name }}
   
 
   # devenv passes in paths to binaries it uses (ensuring the supported versions are used)
   # devenv bin that triggered devspace
   # orc setup installs devenv bin so we can fallback to that if needed
-  - name: DEVENV_BIN
+  DEVENV_BIN:
     source: env
     default: devenv
   # devspace bin for running devspace commands (e.g. devspace run build-jsonnet)
   # orc setup installs devspace bin so we can fallback to that if needed
-  - name: DEVENV_DEVSPACE_BIN
+  DEVENV_DEVSPACE_BIN:
     source: env
     default: devspace
   # kind bin for loading images into local dev-environment
   # this var is passed in only when deploying to local dev-environment
-  - name: DEVENV_KIND_BIN
+  DEVENV_KIND_BIN:
     source: env
 
   # This var isn't produced by devenv, but can be produced by user scripts to override the namespace.
-  - name: DEVENV_DEPLOY_NAMESPACE
+  DEVENV_DEPLOY_NAMESPACE:
     source: env
     default: ${DEVENV_DEPLOY_APPNAME}--bento1a
 
-  - name: GH_TOKEN
-    source: command
-    command: gh auth token
-  - name: NPM_TOKEN
-    source: command
-    command: grep -E "registry.npmjs.org(.+)_authToken=(.+)" $HOME/.npmrc | sed 's/.*=//g'
-  - name: APP_VERSION
-    source: command
-    command: make version
-  - name: BOX_REPOSITORY_URL
-    source: command
-    command: yq -r '.storageURL' "$HOME/.outreach/.config/box/box.yaml"
+  GH_TOKEN: $(yq -r '.["github.com"].oauth_token' "$HOME/.config/gh/hosts.yml")
```

Oops, wrong place :D

```suggestion
  GH_TOKEN: $(gh auth token)
```